thirdparty/runtime_environments.go

```diff
@@ -472,22 +477,34 @@ func (c *RuntimeEnvironmentsClient) GetImageInfo(ctx context.Context, imageID st
 
 // buildImageEventEntry make an ImageEventEntry given an ImageDiffChange.
 func buildImageEventEntry(diff ImageDiffChange) (*ImageEventEntry, error) {
-	var action ImageEventEntryAction
+	var eventAction ImageEventEntryAction
```

Comment:
nit: I'd prefer to use switch statements for here and L491 for large (>4) conditional blocks

Reply:
Couldn't change the first one since it operates on two variables (I believe), but updated the second one!